torch/_torch_docs.py

```diff
@@ -5278,6 +5278,45 @@ def merge_dicts(*dicts):
             [ 8,  9]])
 """)
 
+add_docstr(torch.nan_to_num,
+           r"""
+nan_to_num(input, nan=0.0, posinf=None, neginf=None, *, out=None) -> Tensor
+
+Replace NaN with zero and infinity with large finite numbers (default behaviour)
+or with the numbers defined by the user using the nan, posinf and/or neginf keywords.
+
+If x is inexact, NaN is replaced by zero or by the user defined value in nan keyword,
+infinity is replaced by the largest finite floating point values representable by x.dtype
+or by the user defined value in posinf keyword and -infinity is replaced by the most
+negative finite floating point values representable by x.dtype or by the user defined
+value in neginf keyword.
+
+Args:
+    {input}
+    nan (float or optional): Value to be used to fill NaN values.
+    If no value is passed then NaN values will be replaced with 0.0.
+    posinf (float or optional): Value to be used to fill positive infinity values.
```

Comment:
posinf (Number, optional): if a Number, the value to replace positive infinity values with. If None, positive infinity is replaced with the greatest finite value representable by :attr:\`input\`'s dtype. Default is None.